spec/permutations.cry

```diff
@@ -0,0 +1,49 @@
+// The functions in this module are derived from the consensus-specs:
+// @see https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/polynomial-commitments.md#helper-functions
+
+module permutations where
+
+// Check if `value` is a power of two integer
```

Comment:
My current habit is to avoid periods at the end of the last sentence --- looks less busy, or at least that is what I've come to be use to.

BUT, I want to follow the conventions, so I'll make this fix. Thank you!!

Reply:
Yeah, my mysterious preference is full sentences in doc comments, skip ending punctuation for inline comments... I am not sure it's rational but I hear you on the busy-ness.